src/tree.hpp

```diff
@@ -132,6 +134,7 @@ namespace NodeGLPK {
             V8CHECK(!host->handle, "object deleted");
             V8CHECK(host->thread.load(), "an async operation is inprogress");
             
+            MemStatsGuard mguard{host->memstats_}; 
```

Comment:
consider moving to helper function that instantiates guards

Reply:
fixed in 25dcef0 (as a macro; because it fits this codebase better)